docs/modules/control/inverted_pendulum_lqr_control/inverted_pendulum_mpc_control.rst

```diff
@@ -0,0 +1,6 @@
+Inverted Pendulum LQR Control
+-----------------------------
+
+Bipedal Walking with modifying designated footsteps, using LQR controller
+
+.. image:: https://github.com/AtsushiSakai/PythonRoboticsGifs/raw/master/Control/InvertedPendulumCart/animation_lqr.gif
```

Comment:
If it is possible, could you add technical documents here to explain the inverted pendulum model, LQR conroller model, etc.